client/js/lounge.js

```diff
@@ -170,6 +170,10 @@ $(function() {
 		$("#sign-in").remove();
 
 		var id = data.active;
+
+		// Set it right away before triggering a click so that "open" event is not emitted
```

Comment:
Maybe this isn't actually a good idea, as loading the page *should (?)* mark the channel as read.